docs/providers/aws/guide/serverless.yml.md

```diff
@@ -408,6 +408,14 @@ functions:
           conditions:
             host: example.com
             path: /hello
+          healthCheck: # optional, can also be set using a boolean value
+            path: / # optional
+            intervalSeconds: 35 # optional
+            timeoutSeconds: 30 # optional
+            healthyThresholdCount: 5 # optional, must equal unhealthyThresholdCount
+            unhealthyThresholdCount: 5 # optional, must equal healthyThresholdCount
```

Comment:
Should values for both properties equal each other? If so, shouldn't we just support `thresholdCount` ?

Reply:
That was actually a misreading of the AWS docs on my part. That limitation does not apply to health checks using the HTTP/HTTPS protocols. I'll correct it.